cargo-dist/src/backend/ci/github.rs

```diff
@@ -116,6 +119,21 @@ impl GithubCiInfo {
             .hosts
             .clone();
 
+        let release_phase = if dist.github_release == GithubReleasePhase::Auto {
+            // If Axo is in use, we want the release to come late
+            // because the release wil lcontain links to Axo URLs
+            // that won't become live until the announce phase.
```

Comment:
I was about to say this is incorrect but oh right the actual release *body* does use the prettier URLs that need an announce to go live (the actual artifacts refer to the already-live artifact set).